pdns/dnsname.hh

```diff
@@ -328,28 +333,37 @@ struct SuffixMatchTree
     if(children.empty()) { // speed up empty set
       if(endNode)
         return &d_value;
-      return 0;
+      return nullptr;
     }
-    return lookup(name.getRawLabels());
+
```

Comment:
Wondering if this is neccesary. I would think that this case is handled by `lookup(std::vector<string>)`

Reply:
Good catch, I don't think this is necessary! I'll update this PR.